covfee/client/tasks/continuous_annotation/index.tsx

```diff
@@ -657,7 +683,7 @@ const ContinuousAnnotationTask: React.FC<Props> = (props) => {
     "https://app.prolific.com/submissions/complete?cc=" +
     props.spec.prolificCompletionCode
 
-  if (args.response.submitted) {
+  if (args.response.submitted && isEntireTaskCompleted) {
```

Comment:
Note. This is important to allow annotators access the task when there were issues.

Reply:
Should we always let the annotators access the task again? For example, if we ask them to mark additional annotators after evaluating the responses?

Reply:
Resolved IRL discussion: keeping the blockage for data privacy reasons.